docs/README.rst

```diff
@@ -85,46 +68,47 @@ and
 
 Download a copy of the ``.synapseConfig`` file, open the file in the
 editor of your choice and edit the
-`username <https://github.com/Sage-Bionetworks/synapsePythonClient/blob/master/synapseclient/.synapseConfig#L8>`__
+`username <https://github.com/Sage-Bionetworks/synapsePythonClient/blob/v2.2.2-rc/synapseclient/.synapseConfig#L8>`__
 and
-`apikey <https://github.com/Sage-Bionetworks/synapsePythonClient/blob/master/synapseclient/.synapseConfig#L9>`__
+`apikey <https://github.com/Sage-Bionetworks/synapsePythonClient/blob/v2.2.2-rc/synapseclient/.synapseConfig#L9>`__
 attributes under the
-`[authentication] <https://github.com/Sage-Bionetworks/synapsePythonClient/blob/master/synapseclient/.synapseConfig#L7>`__
+`[authentication] <https://github.com/Sage-Bionetworks/synapsePythonClient/blob/v2.2.2-rc/synapseclient/.synapseConfig#L7>`__
 section.
 
  Description of config.yml attributes
 
 ::
 
     definitions:
-        synapse_config: "Path to .synapseConfig file"
-        creds_path: "Path to credentials.json file"
-        token_pickle: "Path to token.pickle file"
-        service_acct_creds: "Path to service_account_creds.json file"
+        synapse_config: # path to .synapseConfig file
+        creds_path: # path to credentials.json file
+        token_pickle: # path to token.pickle file (auto-generated)
+        service_acct_creds: # path to service_account_creds.json file
```

Comment:
I worry that the lack of placeholders might be confusing to some.

Reply:
Ah, makes sense. Added placeholders for all the keys in the description of the `config.yml` file.